articles/language/expressions.md

```diff
@@ -540,6 +540,32 @@ concatenation would be expressed as:
 All arrays in Q# are zero-based.
 That is, the first element of an array `a` is always `a[0]`.
 
+:new: Starting with our 0.8 release, we are supporting contextual expressions for range slicing. 
+In particular, range start and end values may be omitted in the context of a range slicing expression. 
```

Comment:
You should say something here about using ellipsis, "...", instead of just leaving the value off.

Reply:
Should I? I had it listed as example, but right now it has no purpose whatsoever, so it seems that talking about it would make much more sense when we have the multidimensional case. I'll add back the example though.